src/main/java/com/dataloom/edm/EdmApi.java

```diff
@@ -197,4 +199,12 @@ Void updateSchema(
     @GET( BASE + IDS_PATH + ENTITY_TYPE_PATH + NAMESPACE_PATH + NAME_PATH )
     UUID getEntityTypeId( @Path( NAMESPACE ) String namespace, @Path( NAME ) String name );
 
+    @PATCH( PROPERTY_TYPE_BASE_PATH + ID_PATH )
```

Comment:
Per our conversation you could use path, but it's not worth it. Fix to converter is more important.